moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1142,6 +1147,7 @@ void MotionPlanningDisplay::onRobotModelLoaded()
                                                                     planning_scene_monitor_->getTFClient()));
   query_start_state_->setUpdateCallback(boost::bind(&MotionPlanningDisplay::scheduleDrawQueryStartState, this, _1, _2));
   query_goal_state_->setUpdateCallback(boost::bind(&MotionPlanningDisplay::scheduleDrawQueryGoalState, this, _1, _2));
+  previous_state_ = ks;
```

Comment:
Small request: can you create a separate PR that renames ``ks`` to ``robot_state``? ks is left over from when the class used to be called KinematicState, but that was changed many years ago. Also, we strongly discourage abbreviations in our [style guide](https://moveit.ros.org/documentation/contributing/code/)

Reply:
Sure, I will do it.